documentation-support/pom.xml

```diff
@@ -49,11 +49,6 @@
             <artifactId>auto-service-annotations</artifactId>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>com.google.auto.value</groupId>
-            <artifactId>auto-value-annotations</artifactId>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
Auto Value isn't removed from the top-level `pom.xml`, as we still [use](https://github.com/PicnicSupermarket/error-prone-support/blob/faa3a6b05501e1da1ce123e17cab54846871e7d0/refaster-support/src/test/java/tech/picnic/errorprone/refaster/AnnotatedCompositeCodeTransformerTest.java#L191-L205) `@AutoAnnotation`.